packages/common/src/utils/formatting.test.ts

```diff
@@ -1423,4 +1424,28 @@ describe('Formatting', () => {
             });
         });
     });
+
+    describe.only('applyDefaultFormatting', () => {
+        it('should handle boolean values', () => {
+            expect(applyDefaultFormat(true)).toBe('true');
+            expect(applyDefaultFormat(false)).toBe('false');
+        });
+        it('should handle falsey values', () => {
+            expect(applyDefaultFormat(undefined)).toBe('-');
+            expect(applyDefaultFormat(null)).toBe('∅');
+            expect(applyDefaultFormat(0)).toBe('0');
+            expect(applyDefaultFormat('')).toBe('');
+        });
+        it('should handle strings', () => {
+            expect(applyDefaultFormat('foo')).toBe('foo');
+        });
+        it('should handle numbers', () => {
+            expect(applyDefaultFormat(1234567)).toBe('1,234,567');
+            expect(applyDefaultFormat(1234)).toBe('1,234');
+            expect(applyDefaultFormat(-1234)).toBe('-1,234');
+        });
+        it('should handle arrays', () => {
+            expect(applyDefaultFormat([1, 2, 3])).toBe('1,2,3');
```

Comment:
I think this is the default `toString`(arrays->string) behavior and not part of the original implementation of `applyDefaultFormat`